Dockerfile

```diff
@@ -5,7 +5,7 @@ RUN apt-get update && apt-get install -y qemu-system-x86 rsync qemu-kvm minicom
     rm -rf /var/lib/apt/lists/* && \
     pip install --upgrade pip && \
     pip install robotframework==3.0 requests==2.4.3 robotframework-requests==0.4.5 pylibftdi==0.15.0 && \
-    npm install --global resin-cli@7.10.6 && \
+    npm install --global balena-cli@9.12.6 && \
     git clone --depth 1  --branch v1.0.0-beta.18 https://github.com/resin-io/etcher.git && cd /etcher && \
```

Comment:
Wow, super-ancient Etcher :rofl: 
But I guess if it still works there's no need to change it.